src/dlgConnectionProfiles.cpp

```diff
@@ -31,8 +31,10 @@
 
 #include "pre_guard.h"
 #include <QtUiTools>
-#include "post_guard.h"
+#include <QSettings>
 #include <sstream>
+#include <../3rdparty/qtkeychain/keychain.h>
```

Comment:
This looks like an odd include, but that was the only way I could get it to work for both qmake and cmake.

Reply:
It *has* to be a system include and not a local one (using `"`...`"`) - strange?